docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -97,93 +159,120 @@ Windows::
+NOTE: The _migration.yaml_ file uses the same configuration options as the Hazelcast CLC. For further information on the options, refer to the https://docs.hazelcast.com/clc/latest/clc-config[Hazelcast CLC documentation^].
 
-NOTE: The `migration.yaml` file used by DMT is in https://docs.hazelcast.com/clc/latest/clc-config[Hazelcast CLC configuration format], i.e., DMT and CLC uses the same configuration format. DMT uses this configuration file to connect to the migration cluster as described in the next step below.
+=== Run the Migration
 
-=== Step 5. Start the Migration using DMT
+Before running the migration, you need the following information:
 
-Go to `bin` in the folder where you extracted the DMT package; there are DMT binaries in the format `dmt_[platform]_[arch]`. Note the binary suitable 
-for your machine; you need to know your operating system and your processor architecture. For `arm`, choose the suitable `arm64` binary, and for Intel, choose the suitable `amd64` binary.
+* Your Operating System
+* Your processor architecture
+* The binary that is suitable for your machine
 
-Run the following command in the folder where you extracted the DMT package, to start the migration.
+You can find DMT binaries, in the format `dmt_[platform]_[arch]`, in the _bin_ folder of the extracted DMT package. Use the ``arm64`` binary for ARM, and the `amd64` binary for Intel.
 
+To run the migration, complete the following steps:
+
+. Open a terminal
+. Navigate to the folder containing the extracted DMT package
+. Enter the following command:
++
 [source,shell]
 ----
-$ ./bin/dmt_[platform]_[arch] --config migration.yaml start migration_config --yes
+./bin/dmt_[platform]_[arch] --config migration.yaml start migration_config --yes
 ----
 
-If you are on macOS, and the above command is rejected by the operating system, click `OK`, go to `Privacy & Security` settings of the machine, and allow the `dmt*`binary to run.
-Then retry and click `Open` on the OS' dialog.
+NOTE: On MacOS, you might need to allow the `dmt*` binary to run. If the command is rejected, go to the *Privacy & Security* settings on your device and update them to allow the binary. After updating the settings, retry the command, and select *Open* when prompted.
+
+You can use the https://docs.hazelcast.com/management-center/latest/getting-started/overviewHazelcast[Management Center^], or the DMT `status` command to track the migration. For further information on the available DMT commands, see the xref:migrate:dmt-command-reference.adoc[DMT Command Reference].
+
+=== Verify the Migrated Data
 
-=== Step 6. Verify the Data in Target Cluster
+You can verify the size of the map in the target cluster in the following ways:
 
-You can verify the size of the map in the target cluster using Hazelcast CLC.
-Run the following CLC commands to see the size of the map, and the value of a random key from the data we put in Step 2.
+* Use the Hazelcast Management Center
```

Comment:
again, verifiying via MC is via checking data structures and their size etc. Or maybe via map browser.

Reply:
Are you saying MC doesn't support this yet? If it is something that the user has to look at and work out themselves from other information, this isn't an ideal method to suggest. 

I also think we'd have to include much more information to explain how to verify using the data structure size/map browser if we leave it in.....

Reply:
Via MC they can check the target map size, use map browser to browse entries. (we can give a link to it here). MC supports this in theory, because as part of DMT effort, there won't be any new UI for migrated data verification.

Reply:
I'd say let's keep it but let's elaborate how they can use MC:

1. Check target map size in storage section (give a link)
2. Check map entries via map browser (give a link)